### ISSUE TITLE
flutter_inappwebview: ^6.0.0-beta.24+1 breaking in iOS17

### ISSUE THREAD

**Issue by anarnoli** [pichillilorenzo/flutter_inappwebview#1799]
- [x] I have read the [Getting Started](https://inappwebview.dev/docs/intro/) section
- [x] I have already searched for the same problem

## Environment

| Technology      | Version       |
|-----------------| ------------- |
| Flutter version |     3.13.5          |
| Plugin version  |     ^6.0.0-beta.24+1          |
| Android version |               |
| iOS version     |    iOS 17           |
| macOS version   |               |
| Xcode version   |       15        |

Device information: 

## Description
flutter_inappwebview breaking iOS build after upgrade to iOS17. Here is error message

```
Launching lib/main.dart on iPhone 15 Pro in debug mode...
main.dart:1
Xcode build done.                                           23.4s
Failed to build iOS app
Error (Xcode): type argument 'nw_proxy_config_t' (aka 'struct nw_proxy_config *') is neither an Objective-C object nor a block type
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.0.sdk/System/Library/Frameworks/WebKit.framework/Headers/WKWebsiteDataStore.h:119:46

Parse Issue (Xcode): Could not build module 'WebKit'
/build/ios/Debug-iphonesimulator/flutter_inappwebview/flutter_inappwebview.framework/Headers/flutter_inappwebview-Swift.h:1162:8

Uncategorized (Xcode): Command SwiftEmitModule failed with a nonzero exit code

Could not build the application for the simulator.
Error launching application on iPhone 15 Pro.
Exited
```

**Expected behavior:** 
Successful build and launch on iPhone 15 Pro emulator.

**Current behavior:** 
Build failed

**Comment by anchao-lu** [comment]
same issue

**Comment by LarsRefsgaard** [comment]
https://github.com/pichillilorenzo/flutter_inappwebview/issues/1783

**Comment by anchao-lu** [comment]
> #1735

I know it. Maybe it's a viable approach, but I don't think it's a perfect one

**Comment by LarsRefsgaard** [comment]
Indeed, but it tracks the same issue, so no need to duplicate it. 

> > #1735
> 
> I know it. Maybe it's a viable approach, but I don't think it's a perfect one

**Comment by anarnoli** [comment]
Following [#1735](https://github.com/pichillilorenzo/flutter_inappwebview/issues/1735), I am able to workaround for now:

1. Open file 
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Cryptexes/OS/System/Library/Frameworks/WebKit.framework/Headers/WKWebsiteDataStore.h

2. Change 170000 to 180000 as shown below: 
#if ((TARGET_OS_OSX && __MAC_OS_X_VERSION_MAX_ALLOWED >= 140000) \
    || ((TARGET_OS_IOS || TARGET_OS_MACCATALYST) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 180000) \
    || (TARGET_OS_WATCH && __WATCH_OS_VERSION_MAX_ALLOWED >= 100000) \
    || (TARGET_OS_TV && __TV_OS_VERSION_MAX_ALLOWED >= 180000))

3. Save file and Restart app, you should be fine.

**Comment by pichillilorenzo** [comment]
Released new version `5.8.0` and `6.0.0-beta.25` with the fix!
